conbench/__init__.py

```diff
@@ -33,16 +33,11 @@
 conbench.logger.setup(level_stderr="DEBUG", level_file=None, level_sqlalchemy="WARNING")
 log = logging.getLogger(__name__)
 
-# This is going to be an application-global singleton (in the webapp, not the
-# CLI).
-metrics = None
```

Comment:
I removed this again, it was premature thinking.

Reply:
Nice, I'm scared of globals. :)